src/MobxPromise.ts

```diff
@@ -143,6 +150,17 @@ export class MobxPromiseImpl<R> {
 
     @computed get result(): R | undefined {
         // checking status may trigger invoke
+        /*@ts-ignore*/
+        if (this.isComplete === false) {
+
+            if (localStorage.debugMP) {
+                debugger;
+            }
```

Comment:
Is this `debugger` on purpose? Do we need this because source code is not mapped?